dlt-connector/src/client/BackendClient.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import { gql, GraphQLClient } from 'graphql-request'
+
+import { CONFIG } from '@/config'
+import { CommunityDraft } from '@/graphql/input/CommunityDraft'
+import { logger } from '@/logging/logger'
+import { LogError } from '@/server/LogError'
+
+const communityByForeign = gql`
+  query ($foreign: Boolean) {
+    community(foreign: $foreign) {
+      uuid
+      foreign
+      creationDate
+    }
+  }
```

Comment:
Ah i see here is you usecase to request `community` with only a boolean as parameter.
But i think it will be necessary to split the requests from frontend/admininterface and from dlt-connector in different queries/graphql-methods because of security reasons for different users with different roles and rights.

This will also solve the problem to avoid using the 1st parameter `communityIdentifier` as type boolean, right?

Reply:
> This will also solve the problem to avoid using the 1st parameter communityIdentifier as type boolean, right?

I have changed it